docs/docs/30-how-to-guides/14-working-with-stages.md

```diff
@@ -0,0 +1,556 @@
+          id: 5535a484-bbd0-4f12-8cf4-be2c8e0041c9
+          phase: Pending
+          startTime: "2024-07-15T22:13:34Z"
+      phase: Succeeded
+  observedGeneration: 1
+  phase: Steady
+```
+
+## Interacting with Stages
+
+Kargo provides tools to manage Stages using either the Kargo Dashboard or the
+Kargo CLI. This section explains how to handle Stages effectively through both interfaces.
+
+Kargo simplifies managing `Stage`s through the Kargo Dashboard or the Kargo CLI. This section
+explains how to effectively perform key actions—like creating, deleting, refreshing Stages and
+verifying it's Freight—using both interfaces.
+
+### Creating a Stage
+
+<Tabs groupId="create-stage">
+  <TabItem value="ui" label="Using the Kargo Dashboard" default>
+
+    Navigate to your Project dashboard and locate the action menu in the right corner of the pipeline:
+
+    ![create-stage](../../static/img/create-stage.png)
+
+    Click the magic wand icon to open the dropdown, then select <Hlt>Create Stage</Hlt>.
+    A form will appear to input details for the new `Stage`:
+
+    ![create-stage](../../static/img/create-stage-2.png)
+
+    Complete the form with the necessary details and submit it.
+    The new `Stage` will be added to the pipeline, connected to other
+    `Stage`s based on your configuration:
+
+    ![create-stage](../../static/img/create-stage-3.png)
+  </TabItem>
+
+  <TabItem value="cli" label="Using the Kargo CLI">
+
+    Define the `Stage` in a YAML file, for example:
+
+    ```yaml
+    apiVersion: kargo.akuity.io/v1alpha1
+    kind: Stage
+    metadata:
+      name: <stage-name>
+      namespace: <project-name>
+    spec:
+      ### Add your Stage specifications here
+    ```
+
+    Save the file and run:
+
+    ```shell
+    kargo create -f <stage-filename>
+    ```
+
+    Example Output:
+    ```shell
+    stage.kargo.akuity.io/<stage-name> created
+    ```
+
+    Verify the creation by listing the `Stage`:
+
+    ```shell
+    kargo get stage <stage-name> --project <project-name>
+    ```
+
+    Example Output:
+    ```shell
+    NAME    SHARD   CURRENT FREIGHT   HEALTH   PHASE  AGE
+    test                                              70s
+    ```
+  </TabItem>
+</Tabs>
+
+### Deleting a Stage
+
+<Tabs groupId="delete-stage">
+  <TabItem value="ui" label="Using the Kargo Dashboard" default>
+
+    Select the `Stage` you want to remove and click <Hlt>Delete</Hlt> in the top-right corner of the pop-up window:
+
+    ![delete-stage](../../static/img/delete-stage.png)
+  </TabItem>
+
+  <TabItem value="cli" label="Using the Kargo CLI">
+    To delete a `Stage` using the CLI, run:
+
+    ```shell
+    kargo delete stage --project <project-name> <stage-name>
+    ```
+
+    Example Output:
+    ```shell
+      stage.kargo.akuity.io/<stage-name> deleted
+    ```
+  </TabItem>
+</Tabs>
+
+### Refreshing a Stage
+
+Refreshing ensures the `Stage` reflects the latest updates,
+including changes in `Freight` or configuration.
```

Comment:
I'm not sure I'd describe it this way. It forces the controller to reconcile the Stage, which means:

1. Check for recently completed Promotions
2. Cascade reconciliation to any in-progress Promotion
3. Run health checks
4. Verify or reverify the current Freight
5. Decide whether the next pending Promotion (if any) may now proceed
6. If auto-promotion is enabled, look for new Freight and create corresponding Promotion(s)

Reply:
idk if you actually need to include this level of detail, but my point is only that when you refresh, real work happens. It's not just a trivial update of the status, which is how I read this.